crates/copybook-dialect/claude.md

```
@@ -1,16 +1,16 @@
 <!-- SPDX-License-Identifier: AGPL-3.0-or-later -->
 # claude.md
 
-## Scope
-- Dialect contract for ODO min_count semantics.
-- Layer: **Schema**.
-- Workspace package `copybook-dialect` (see `Cargo.toml`).
+## Scope
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)